content/develop/connect/clients/php.md

```diff
@@ -0,0 +1,314 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Connect your PHP application to a Redis database
+linkTitle: PHP
+title: PHP guide
+weight: 6
+---
+
+[`Predis`](https://github.com/predis/predis) is the recommended [PHP](https://php.net/)
+client for Redis. 
+The sections below explain how to install `Predis` and connect your application to a Redis database.
+
+{{< note >}}Although we provide basic documentation for `Predis`, it is a third-party
+client library and is not developed or supported directly by Redis.
+{{< /note >}}
+
+`Predis` requires a running Redis or
+[Redis Stack]({{< relref "/operate/oss_and_stack/install/install-stack/" >}}) server.
+See [Getting started]({{< relref "/operate/oss_and_stack/install/" >}}) for Redis installation
+instructions.
+
+## Install
+
+Use [Composer](https://getcomposer.org/) to install the `Predis` library
+with the following command line:
+
+```bash
+composer require predis/predis
+```
+
+## Connect
+
+Connect to a locally-running server on the standard port (6379)
+with the following code:
+
+```php
+<?php
+
+require 'vendor/autoload.php';
+
+use Predis\Client as PredisClient;
+
+$r = new PredisClient([
+                'scheme'   => 'tcp',
```

Comment:
All of this are default values, can be omitted for simplicity

Reply:
@vladvildanov I've removed them. There's another example in the page that illustrates them, so as you say, best to use the simplest example to start off.